### DISCUSSION TITLE
Thanks!

### DISCUSSION THREAD

**Discussion by Syfoon** [Psifi-Solutions/csrf-csrf#16] [General]
Hi Psifi,

Thanks for creating csrf-csrf! When revisiting an old project I saw the message that csurf was deprecated… I felt really lucky finding this one:-) It was easy to replace csurf with your csrf-csrf solution. Love the config simplicity. Keep up the good work!

**Comment by psibean** [comment]
No problem! Glad it's helping! It's definitely picking up some traction on npm lately.

I'll have more open source projects coming up in the future (not necessarily express related) under Psifi Solutions too, so keep an eye out!
